pipeline/api/cloud.py

```diff
@@ -12,7 +12,6 @@
 import httpx
 import requests
```

Comment:
I think `requests` is still referenced in `_get_raise_for_status(...)` so if this is updated it can be removed entirely